api/src/services/competenciesService.ts

```diff
@@ -0,0 +1,10 @@
+import db from './db';
+
+export const listCompetencies = async (limit: number, offset: number) => {
+  const competencies = await db('competencies')
+    .select('id', 'label', 'description')
```

Comment:
The competencies tables has other columns.  Should the function also retrieve those columns?

Reply:
Yes. Please let me know what are those columns.

Reply:
created_at

Reply:
We deleted created_at.

Reply:
I agree. I don't think we need to retrieve those columns right now.  I noticed that the ReflectionsService selected for those columns.